stl/inc/limits

```diff
@@ -1116,6 +1119,20 @@ _NODISCARD int _Checked_x86_x64_popcount(const _Ty _Val) noexcept {
 }
 #endif // _HAS_POPCNT_INTRINSICS
 
+#if defined(_M_ARM64) && !defined(_M_CEE_PURE) && !defined(__CUDACC__) && !defined(__INTEL_COMPILER) \
+    && !defined(__clang__) // TRANSITION, LLVM-51488

```

Comment:
`<arm64_neon.h>` was guarded by `defined(_M_ARM64) && !defined(_M_ARM64EC)`, but `_M_ARM64EC` isn't being considered here. (I still have an extremely limited understanding of how intrinsics interact with ARM64EC.)

I recommend following the pattern in `<cmath>`, where we first define the "have cool intrinsics" macro, and then use it to determine whether to include the necessary header. This both prevents the conditions from getting out of sync, and avoids dragging in the header for platforms where it won't even be used:

https://github.com/microsoft/STL/blob/f75c7f596c7b491168fefb5eff5164ab7ae36867/stl/inc/cmath#L15-L23